microsoft-edge/webview2/how-to/winrt-from-js.md

```diff
@@ -0,0 +1,336 @@
+---
+title: Call native-side WinRT code from web-side code
+description: How to pass WinRT host objects to JavaScript using the AddHostObjectToScript API for WebView2 apps.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/02/2022
+---
+# Call native-side WinRT code from web-side code
+
+Your web-side JavaScript code can access the methods and properties of your WinRT host app, by using the _WebView2 WinRT JS Projection tool_.  (For short, this is sometimes called the _WV2WinRT_ tool.)  For more information about why you'd want to do this, see the introduction of [Call native-side code from web-side code](hostobject.md).
```

Comment:
I think "wv2winrt" is now the official name of the tool. It matches the pattern and casing for cppwinrt and rnwinrt.